src/decorator-annotator.ts

```diff
@@ -148,7 +148,8 @@ export class DecoratorClassVisitor {
    * This method looks for the place where the value that is associated to
```

Comment:
Ah, I see, thanks for explaining.  This is well-documented in the file, I just didn't see it in the context, so please disregard this comment.